pose-detection/src/movenet/README.md

```diff
@@ -0,0 +1,97 @@
+# MoveNet
+
+MoveNet is an ultra fast and accurate model that detects 17 keypoints of a body.
+The model is offered on TF.Hub with two variants,
+[Lightning](https://tfhub.dev/google/movenet/singlepose/lightning/3) and
+[Thunder](https://tfhub.dev/google/movenet/singlepose/thunder/3). Lightning is
+intended for latency-critical application, and Thunder is intended for
+applications that require high accuracy. Both models run faster than real time
+(30+ FPS) on most modern desktops and laptops, which proves crucial for live
+fitness, health, and wellness applications. Please try it out using the live
+[demo](https://storage.googleapis.com/tfjs-models/demos/pose-detection/index.html?model=movenet).
+
+--------------------------------------------------------------------------------
+
+## Table of Contents
+
+1.  [Installation](#installation)
+2.  [Usage](#usage)
+3.  [Example Code and Demos](#example-code-and-demos)
+
+## Installation
+
+Please follow the instructions in the Pose API
+[README](https://github.com/tensorflow/tfjs-models/blob/master/pose-detection/README.md)
```

Comment:
Can you link to the Installation section here?

https://github.com/tensorflow/tfjs-models/blob/master/pose-detection/README.md#installation